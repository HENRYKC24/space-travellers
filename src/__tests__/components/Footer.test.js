import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

it('should render correctly', () => {
  const tree = renderer.create(<BrowserRouter><Footer /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
