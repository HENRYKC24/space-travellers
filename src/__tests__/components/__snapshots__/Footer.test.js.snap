// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<footer
  className="footer"
>
  <p
    className="footer-text"
  >
    Created with ðŸ’– by Henry and Victor at Microverse Inc., US.
  </p>
</footer>
`;
